### PR TITLE
[istio] USAGE.md renew

### PR DESCRIPTION
**Pull request by apolovov**
## Description
USAGE.md refreshing

## Why do we need it, and what problem does it solve?
Some instructions were rethought and apiVersions were outdated.

## Changelog entries
```changes
section: istio
type: chore
summary: USAGE.md documentation refreshing
```

### CODE REVIEW

File: ee/modules/110-istio/docs/USAGE.md [hunk 1]
```diff
@@ -473,6 +464,56 @@ spec:
  rules: [{}]
 ```
 
+## Setting up federation for two clusters using the IstioFederation CR
+
+сluster A:
```
Comment (apolovov) [inline ee/modules/110-istio/docs/USAGE.md:469]:
```suggestion
cluster A:
```

File: ee/modules/110-istio/docs/USAGE.md [hunk 2]
```diff
@@ -473,6 +464,56 @@ spec:
...
+apiVersion: deckhouse.io/v1alpha1
+kind: IstioFederation
+metadata:
+  name: cluster-b
+spec:
+  metadataEndpoint: https://istio.k8s-b.example.com/metadata/
+  trustDomain: cluster-b.local
+```
+
+сluster B:
```
Comment (apolovov) [inline ee/modules/110-istio/docs/USAGE.md:481]:
```suggestion
cluster B:
```

File: ee/modules/110-istio/docs/USAGE_RU.md [hunk 3]
```diff
@@ -473,6 +465,56 @@ spec:
  rules: [{}]
 ```
 
+## Устройство федерации из двух кластеров с помощью CR IstioFederation
+
+сluster A:
```
Comment (apolovov) [inline ee/modules/110-istio/docs/USAGE_RU.md:470]:
```suggestion
cluster A:
```

File: ee/modules/110-istio/docs/USAGE_RU.md [hunk 4]
```diff
@@ -473,6 +465,56 @@ spec:
...
+apiVersion: deckhouse.io/v1alpha1
+kind: IstioFederation
+metadata:
+  name: cluster-b
+spec:
+  metadataEndpoint: https://istio.k8s-b.example.com/metadata/
+  trustDomain: cluster-b.local
+```
+
+сluster B:
```
Comment (apolovov) [inline ee/modules/110-istio/docs/USAGE_RU.md:482]:
```suggestion
cluster B:
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [2771](https://github.com/deckhouse/deckhouse/pull/2771) to the branch [release-1.38](https://github.com/deckhouse/deckhouse/tree/release-1.38) successful!

**Pull request merged**